templates/sag-db-oracle/README.md

```diff
@@ -18,8 +18,7 @@ With this template you can create user, database, and webMethods database schema
 
 ## Running as a standalone Composite Template
 
-Run this command to install Database Component Configurator on `local` 10.2 managed node and create
-webMethods `webm` database, `webm` user and all schemas for 10.2 release on `oracle` database host connecting with `system` user and `oracle` password:
+To install Database Component Configurator on a node with alias `local` and version 10.2 and create a webMethods database, named `webm` with credentials `webm`/`webm` and all database schemas for version 10.2 on an Oracle database host connecting with `system` /`oracle`:
 
 ```bash
 sagcc exec templates composite apply sag-db-oracle
```

Comment:
sagcc exec templates composite apply sag-db-oracle
   release=10.1 repo.product=products-10.1 repo.fix=fixes-10.1 nodes=local \
   db.host=oracle db.admin.username=system db.admin.password=oracle \
   db.name=webm db.username=webm db.password=webm \
   db.products=[IS,MWS]
   --sync-job --wait 360